reference/recency-surveillance.v1.json

```diff
@@ -293,10 +312,12 @@
                         ]
                       },
                       "accompanied_by_other": {
+                        "description": "Only filled if accompanied_by is other",
                         "type": "string",
                         "default": "text"
                       },
                       "accompanied_by": {
+                        "description": "If the client is a child below 12 years",
                         "type": "string",
                         "enum": [
                           "GRANDPARENT",
```

Comment:
@ODORA0  there's also these capital cases which shouldn't be. Unless its an abbreviation like PITC.

Reply:
@mugumedavid That's OK, I'll change all of them. The time I made these changes I got them directly as they were in the EMR

Reply:
@mugumedavid Done